<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
  <meta http-equiv="Content-Style-Type" content="text/css">
  <title>Hackers Guide to the Microbiome</title>
  <meta name="Author" content="Richard Sprague">
  <meta name="CreationTime" content="2015-07-23T11:45:14Z">
  <meta name="ModificationTime" content="2015-07-23T11:45:14Z">
  <meta name="Generator" content="Cocoa HTML Writer">
  <meta name="CocoaVersion" content="1348.17">
  <style type="text/css">a:link {text-decoration:none;}
a:visited {text-decoration:none;}
a:hover {text-decoration:none;}

    p.p1 {margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Georgia}
    p.p2 {margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Georgia; min-height: 14.0px}
    p.p3 {margin: 0.0px 0.0px 0.0px 0.0px; text-align: right; font: 12.0px Georgia}
    p.p4 {margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 12.0px Georgia; min-height: 14.0px}
    p.p5 {margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 12.0px Georgia}
    p.p7 {margin: 0.0px 0.0px 0.0px 0.0px; text-indent: 36.0px; font: 12.0px Georgia}
    p.p8 {margin: 0.0px 0.0px 0.0px 0.0px; text-indent: 36.0px; font: 12.0px Georgia; min-height: 14.0px}
    p.p13 {margin: 0.0px 0.0px 0.0px 0.0px; text-indent: 18.0px; font: 12.0px Georgia}
    p.p14 {margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Georgia; color: #0b1480}
    p.p15 {margin: 0.0px 0.0px 0.0px 0.0px; text-indent: 36.0px; font: 12.0px Georgia; color: #252525}
    p.p16 {margin: 0.0px 0.0px 0.0px 0.0px; text-indent: 18.0px; font: 12.0px Georgia; min-height: 14.0px}
    p.p17 {margin: 0.0px 0.0px 0.0px 36.0px; font: 14.0px Cochin}
    p.p19 {margin: 0.0px 0.0px 0.0px 36.0px; font: 14.0px Cochin; min-height: 17.0px}
    p.p20 {margin: 0.0px 0.0px 0.0px 36.0px; font: 16.0px Arial; color: #2f2f2f; -webkit-text-stroke: #2f2f2f}
    p.p21 {margin: 0.0px 0.0px 0.0px 0.0px; text-indent: 36.0px; font: 12.0px Georgia; -webkit-text-stroke: #2f2f2f; min-height: 14.0px}
    p.p22 {margin: 0.0px 0.0px 0.0px 0.0px; text-indent: 36.0px; font: 12.0px Georgia; -webkit-text-stroke: #2f2f2f}
    span.s1 {text-decoration: underline}
    span.s2 {font-kerning: none}
    span.s3 {font-kerning: none; color: #000000}
    span.s4 {text-decoration: underline ; font-kerning: none}
    span.s5 {text-decoration: underline ; font-kerning: none; color: #0b1480}
    span.s6 {font: 9.0px Georgia; text-decoration: underline ; vertical-align: 3.6px; font-kerning: none; color: #0b1480}
    span.Apple-tab-span {white-space:pre}
    table.t1 {border-collapse: collapse}
    table.t2 {background-color: #f9f9f9; margin: 14.0px 0.0px 14.0px 0.0px; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; border-collapse: collapse}
    td.td1 {padding: 0.0px 5.0px 0.0px 5.0px}
    td.td2 {width: 54.8px; background-color: #ffe75f; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td3 {width: 31.8px; background-color: #b3dec0; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td4 {width: 32.8px; background-color: #bbbfe0; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td5 {width: 35.8px; background-color: #f8b7d3; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td6 {width: 77.8px; background-color: #b0b0b0; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td7 {width: 31.8px; background-color: #f2f2f2; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td8 {width: 762.8px; background-color: #f2f2f2; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td9 {width: 192.8px; background-color: #f2f2f2; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td10 {width: 184.8px; background-color: #f2f2f2; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td11 {width: 166.8px; background-color: #f2f2f2; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td12 {width: 181.8px; background-color: #f2f2f2; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td13 {width: 41.8px; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td14 {width: 138.8px; background-color: #ffe75f; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td15 {width: 31.8px; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td16 {width: 140.8px; background-color: #b3dec0; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td17 {width: 122.8px; background-color: #b3dec0; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td18 {width: 32.8px; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td19 {width: 136.8px; background-color: #b3dec0; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td20 {width: 122.8px; background-color: #b0b0b0; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td21 {width: 136.8px; background-color: #b0b0b0; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td22 {width: 136.8px; background-color: #ffe75f; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td23 {width: 140.8px; background-color: #ffe75f; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td24 {width: 122.8px; background-color: #bbbfe0; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td25 {width: 136.8px; background-color: #bbbfe0; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td26 {width: 122.8px; background-color: #f8b7d3; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td27 {width: 68.8px; background-color: #f2f2f2; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td28 {width: 201.8px; background-color: #f2f2f2; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td29 {width: 84.8px; background-color: #f2f2f2; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td30 {border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td31 {width: 188.8px; background-color: #f2f2f2; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td32 {width: 201.8px; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td33 {width: 84.8px; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td34 {width: 188.8px; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td35 {width: 285.8px; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
    td.td36 {width: 298.8px; border-style: solid; border-width: 1.0px 1.0px 1.0px 1.0px; border-color: #aaaaaa #aaaaaa #aaaaaa #aaaaaa; padding: 2.8px 5.6px 2.8px 5.6px}
  </style>
</head>
<body>
<div style="width: 600px; margin: 0 auto">

<table cellspacing="0" cellpadding="0" class="t1">
  <tbody>
    <tr>
      <td valign="top" class="td1">
        <p class="p1"><a id="doc11"></a>Richard Sprague</p>
        <p class="p1">PO Box 1523</p>
        <p class="p1">Mercer Island</p>
        <p class="p1">WA</p>
        <p class="p1">98040</p>
        <p class="p1">Sprague@ensembio.com</p>
        <p class="p2"><br></p>
        <p class="p2"><br></p>
      </td>
      <td valign="top" class="td1">
        <p class="p3">9,800 words</p>
      </td>
    </tr>
  </tbody>
</table>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p4"><br></p>
<p class="p5">Hackers Guide to the Microbiome</p>
<p class="p4"><br></p>
<p class="p5">by Richard Sprague</p>
<hr>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<h1 style="margin: 0.0px 0.0px 24.0px 0.0px; text-align: center; font: 24.0px Georgia"><b><a id="doc31"></a>Contents</b></h1>
<p class="p7"><a href="#doc11">Title Page</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc11">?</a></span></p>
<p class="p7"><a href="#doc31">Contents</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc31">?</a></span></p>
<p class="p7"><a href="#doc30">Foreword</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc30">?</a></span></p>
<p class="p7"><a href="#doc4">Chapter One - The Science</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc4">?</a></span></p>
<p class="p7"><a href="#doc71">Intro</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc71">?</a></span></p>
<p class="p7"><a href="#doc6">Microbes everywhere!</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc6">?</a></span></p>
<p class="p7"><a href="#doc35">In the air</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc35">?</a></span></p>
<p class="p7"><a href="#doc41">Underground</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc41">?</a></span></p>
<p class="p7"><a href="#doc37">A crash course in microbiology</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc37">?</a></span></p>
<p class="p7"><a href="#doc42">A crash course in botany</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc42">?</a></span></p>
<p class="p7"><a href="#doc66">What are they doing?</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc66">?</a></span></p>
<p class="p7"><a href="#doc70">Studying the Human Microbiome</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc70">?</a></span></p>
<p class="p7"><a href="#doc38">Chapter Two - The Technology</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc38">?</a></span></p>
<p class="p7"><a href="#doc64">Studying the Microbiome</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc64">?</a></span></p>
<p class="p7"><a href="#doc40">The old way: culturing</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc40">?</a></span></p>
<p class="p7"><a href="#doc39">16s rRNA</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc39">?</a></span></p>
<p class="p7"><a href="#doc43">Gene sequencing</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc43">?</a></span></p>
<p class="p7"><a href="#doc44">Bioinformatics</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc44">?</a></span></p>
<p class="p7"><a href="#doc45">Chapter Three - How to Analyze Your Microbiome</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc45">?</a></span></p>
<p class="p7"><a href="#doc47">How to get your data</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc47">?</a></span></p>
<p class="p7"><a href="#doc46">The basics: web tools</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc46">?</a></span></p>
<p class="p7"><a href="#doc56">Analyze with Excel</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc56">?</a></span></p>
<p class="p7"><a href="#doc72">Comparing samples</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc72">?</a></span></p>
<p class="p7"><a href="#doc51">Chapter Four - Organisms to Watch</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc51">?</a></span></p>
<p class="p7"><a href="#doc52">Good and Bad</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc52">?</a></span></p>
<p class="p7"><a href="#doc75">Gut Guardians</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc75">?</a></span></p>
<p class="p7"><a href="#doc49">Chapter Five - Next Steps</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc49">?</a></span></p>
<p class="p7"><a href="#doc50">Fermented food</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc50">?</a></span></p>
<p class="p7"><a href="#doc59">Annnotated References</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc59">?</a></span></p>
<p class="p7"><a href="#doc29">Endnotes</a><span class="s1"><span class="Apple-tab-span">	</span><a href="#doc29">?</a></span></p>
<hr>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<h1 style="margin: 0.0px 0.0px 24.0px 0.0px; text-align: center; font: 24.0px Georgia"><b><a id="doc30"></a>Foreword</b></h1>
<p class="p7">This book is for curious people with little or no knowledge of biology, but who have a reasonable level of comfort with computer software, either as programmer, or as an end-user comfortable with the basics of installing and running common applications (Excel).</p>
<p class="p8"><br></p>
<p class="p7">The book is divided into four main sections:</p>
<p class="p8"><br></p>
<p class="p7">1. The Science: an introduction to the field of the microbiome. What makes it so interesting? How do scientists go about studying it? How much is known today and what is likely to be discovered in the future?</p>
<p class="p8"><br></p>
<p class="p7">2. Technology. What are the instruments and tools available to study the human microbiome?<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p7">3. How to analyze your own microbiome: we'll walk step-by-step through the process of getting and then analyzing data about your own microbiome. Starting with easy-to-understand web tools, we'll walk you through the basics, then show you how to do simple analysis in Excel. For more complicated analysis, we'll introduce simple (free) programming tools and show step-by-step how to use them to study your own microbiome as well as compare it to others.</p>
<p class="p8"><br></p>
<p class="p7">4. Organisms to watch: Once you've begun the process of studying your own microbes, you'll want to know about some of the common ones and what they do. We'll walk through the basics and show you how to design experiments that let you watch the progress over time.</p>
<p class="p8"><br></p>
<p class="p7">5. Next steps. The information in this book will introduce you to the basics, but once you see how fascinating the microbiome can be, no doubt you'll want to go further. We'll show you how to begin growing your own favorite microbes, and then give you a taste of some of the tools that professional biologists use to analyze in much more detail.</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<hr>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 24.0px Georgia"><b><a id="doc4"></a>Chapter One: The Science</b></h1>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 24.0px Georgia; min-height: 27.0px"><br></h1>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc71"></a>Intro</b></h1>
<p class="p7">I'm a software guy, and when I think about what it means to be human, I think first of course of all the various "software" we are made of: our language, culture, education, behaviors -- all things that seem to us to be flexible, even arbitrary, but which define us as individuals. But as with computers, the software can only run as well as the hardware, so I'm interested in that too: what is the CPU, the memory, the graphics card? and how can you expand or make it better and faster? What should you do to maintain your hardware?</p>
<p class="p8"><br></p>
<p class="p7">To learn more about your hardware, you can of course measure various things about yourself: temperature, blood tests, activity levels. New products like the Apple Watch are making this much easier, and in the near future you'll see an explosion of new and better ways to measure your body's hardware and with better, more actionable measurement come many ways to modify and optimize your health along the way. All of these new wearable computing devices recognize that you<span class="Apple-converted-space">  </span>are a unique individual and that we always should treat the patient, not the statistics. So what is the part of you that is fundamentally you?</p>
<p class="p8"><br></p>
<p class="p7">The obvious place to start is with DNA, and our human genes. I was excited when 23andme (and other services) came out about ten years ago, letting each of us inexpensively see our internal hardware, perhaps to learn more about the things that make us unique, and maybe peer into our futures, to the degree that biological traits like susceptibility to various diseases or conditions is hereditary and unchangeable.</p>
<p class="p8"><br></p>
<p class="p7">But after a while I realized how little of what we learn about DNA is actionable. If you find you have the "gene for"<span class="Apple-converted-space">  </span>Alzheimer's disease or Parkinson's, or any of the other traits that have been linked to specific genes, what can you do about it? In nearly every case, the advice is exactly what you should be doing even if you don't have that trait: eat well and get lots of exercise.</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p7"><b>Fortunately, it's not just our DNA</b></p>
<p class="p7">In fact, if you could point a scanner at me and pull out everything, all the "hardware" inside me right now, you'd find a curious fact: although sure, 99% of the weight and size of what you see is human (blood, skin, bones, organs, etc.), only about 10% of the cells are human, and even less than that<span class="Apple-converted-space">  </span>-- only about 1%<span class="Apple-converted-space">  </span>-- of the DNA-storing genes are human.</p>
<p class="p8"><br></p>
<p class="p7">What's the rest? Who am I, if only 1% of the genes inside me are human? The answer is microbes, and as befits something that so outnumbers the "human" part of us, they play a large role in everything about what we do, from our health to our moods and even, as I'll show, to our motivations.</p>
<p class="p8"><br></p>
<p class="p7"><b>Microbes are older than any of us</b></p>
<p class="p7">We tend to think of the invisible, single-celled microbes as "primitive", not nearly as "advanced" as we humans, with our marvelous brains and ability to transform the earth with airplanes and skyscrapers and nuclear reactors and all the rest. But that's what we would think, wouldn't we? In fact, the microbes are everywhere, literally everywhere on earth, in the sky, even deep underground. We can't go anywhere without encountering microbes because, well, there are even microbes on and inside us. Humans can't survive without microbes. So what does it even mean to say we're "better" or more "advanced" than they are?</p>
<p class="p8"><br></p>
<p class="p7">Before the microscope, people didn't even know that microbes existed. Similarly, until the advent of large-scale gene sequencing machines in the past ten years, almost nothing was known about the amazing ubiquity and resilience of microbes.</p>
<p class="p8"><br></p>
<p class="p7">Yes, they're single-celled and yes many of their cellular functions seem more straightforward than the functions of a multi-cellular creature like us, but it would be a mistake to assume that means microbes -- collectively -- are less influential, and certainly it would be a big error to assume they are less important. Fact is, these organisms have been around, breathing, eating, multiplying, for billions of years, often in pretty much the same form that they are today. These things have survived every imaginable planetary condition from volcanoes to the depths of the ocean to the inside of nuclear reactors. Global Warming means nothing to these guys, who have seen and thrived all over the earth since the day life first appeared.</p>
<p class="p8"><br></p>
<p class="p7"><b>What they want</b></p>
<p class="p7">Because they have such a huge advantage over us, in lifespan (each microbe is an exact divided copy of itself, going back a zillion generations), in speed of replication (they can double in just a few minutes under the right conditions),<span class="Apple-converted-space">  </span>and ubiquity (as I said, cellwise they far outnumber us), they can afford to colonize every new imaginable environment.<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p7">And that's what they do, every time a new frontier opens to them. The moment of your birth, for example, when you left the (mostly) sterile conditions of your mother's womb, they immediately flooded all over your skin, mostly coming from your mother, and in that fresh environment they used their first-mover advantage to get a stronghold that in many cases lasts your entire life. Many (most?) of the microbes that matter arrived inside you this way, originally, and many of them are still there today, decades, even half a century or more later.<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p7">To survive, they need one thing: something to eat. Being so tiny, they don't need much, and they mostly eat things that you (and other larger creatures) weren't interested in anyway. (Or, since they were here first, it's probably more accurate to say that you and I must live on the foods that they don't want. A cheeseburger is only food for you because you snatch it faster than they do. Leave it outside for a while and they'll get it eventually).</p>
<p class="p8"><br></p>
<p class="p7"><b>Who's in charge?</b></p>
<p class="p7">Collectively, the multitude species of microbes will eat just about everything, but individually each species has its preferences. When they're outside the body, as I said, they can "colonize" new territories (like fresh baby skin) to get what they want, but those inside your tummy are at the mercy of whatever it is you decide to put into your mouth.</p>
<p class="p8"><br></p>
<p class="p7">Often, that's not a big deal: many species thrive on the same proteins, carbohydrates, and fats that you do. But some species do better than others with certain types of foods, and this is where the line between your human desires and theirs becomes unclear.</p>
<p class="p8"><br></p>
<p class="p7">Eighty percent of all your brain's outside receptors -- counting all the nerve endings everywhere on your skin -- eighty percent complete their connections in the gut. The main switching grounds, an area called the vagus nerve, does something.<span class="Apple-converted-space">  </span>What? We know very little, but we see some evidence that the purpose -- the reason that not 1% or 10% or 50% but a full 80% of all the receptors go to the gut -- is so the microbes can tell your brain what to do.</p>
<p class="p8"><br></p>
<p class="p7">When you find yourself feeling hungry, ask yourself who is feeling hungry. Scientists have traced that feeling of hunger to changes in certain hormones like leptin, but wait -- why did the leptin levels change in the first place? Could it be that a microbes someplace was manipulating your leptin levels, perhaps by poking that vagus nerve just the right way -- to get your brain to start thinking about whatever food that microbe wants?</p>
<p class="p8"><br></p>
<p class="p7">This isn't as ridiculous as it sounds, the idea that microbes could influence your feelings and desires. Think about a disease like the rabies microbe. Because it spreads through saliva, it can't find new territory unless its host somehow finds itself exchanging saliva -- biting -- another potential host. So guess what a rabies victim can't stop thinking about? Biting a new victim.<span class="Apple-converted-space">  </span>The microbe literally puts a thought into the mind.</p>
<p class="p8"><br></p>
<p class="p7">There are many other examples, so many in fact as to be potentially a bit disturbing when we realize that we humans may be much more at the mercy of tiny microbes than we think.<span class="Apple-converted-space">  </span>Links have been made between microbes and schizophrenia, stress, anxiety, self-grooming, and much more.<span class="Apple-converted-space">  </span>Autism Spectral Disorders, which have always seemed puzzling because of the relationship they seem to have with digestive problems, are also linked to microbes,<span class="Apple-converted-space">  </span>or the lack thereof.</p>
<p class="p8"><br></p>
<p class="p7">Perhaps the most intriguing example is the common parasite Toxoplasma gondii, the strange organism that can only reproduce in the intestines of cats. A parasite seen often in all warm-blooded mammals, it's found in about a third of the global population of humans too.<span class="Apple-converted-space">  </span>It's one of the reasons they tell pregnant women to stay away from cat litter. But here's the interesting part: when a Toxo protozoa infects a mouse, it leaves cysts in the mouse brain that make it attracted to cat urine! Yes, it changes the neurology of a mouse so that it's more likely to end up inside a cat's tummy -- exactly where it can reproduce.</p>
<p class="p8"><br></p>
<p class="p7">Think about this too much, and you'll end up with the obvious question: what other weird microbes are infecting us right now? Can we explain some of our own behaviors this way? Is there a human equivalent of these infections, driving us to do things we "ordinarily" wouldn't do? And maybe these microbes are so ubiquitous, teeming all over us and in our brains, maybe there's no way to even know what "ordinary" or "normal" human behavior is.</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p7"><b>What is health?</b></p>
<p class="p7">There's a new idea in science (well, maybe it's not new, but it's new to me and that's why I'm talking about this). Maybe you can't really think about human hardware without thinking about the whole ecology that goes with it, the various organisms live in and around us and greatly outnumber us.</p>
<p class="p8"><br></p>
<p class="p7">When you think this way, then suddenly "health" takes on a whole new meaning, because we're no longer talking about the status of a single organism -- me -- but rather about the entire function "eco-system" of many, many living things, including<span class="Apple-converted-space">  </span>the "me" that I want to refer to as a human.</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc6"></a>Microbes everywhere!</b></h1>
<p class="p7">Every human body is home to a rich environment of tiny living organisms, trillions of them, with their own genes, their own ways of living and surviving, and their own long histories, permanently intertwined with humanity since the beginning. These microbes inhabit every possible niche inside and outside the body. Covering every inch of skin, lining every part of the nose and ears, they have burrowed deep inside nearly every organ, consuming as well as producing, living in such quantities that together they would weigh somewhere between three and five pounds -- about as much as the brain.</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p7">The word "microbe" refers to any tiny organism that carries its own genetic information for purposes of propagating itself. Far too small to see with the naked eye, dozens could fit inside a typical human cell. The vast majority of microbes are bacteria, though viruses and phages fit this description too, as well as archaea, the extremophile life forms that live and thrive in environments too challenging for bacteria. A typical human will have somewhere between [n and m] individual cells -- blood, skin, neurons, other tissues -- each descended from a single germ cell, the fertilized egg that began dividing itself and multiplying after the moment of conception. By contrast, microbes had to hitch their own rides later, beginning in the womb and turning into a flood at the moment of birth and beyond. But each microbe, like each human cell, remains deeply interconnected with the fates of all the other cells, all the way to the moment of death, and even after that.</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 14.0px Georgia"><b><a id="doc35"></a>In the air</b></h1>
<p class="p7">A team in France, performing experiments on snowflakes, discovered microbes in the sky.</p>
<p class="p13"><img src="Images/Psudomonas syringae.png" width="245" height="180" alt="Image"></p>
<p class="p8"><br></p>
<p class="p7">1</p>
<p class="p8"><br></p>
<p class="p7">These microbes are so numerous that they affect the climate.<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 14.0px Georgia"><b><a id="doc41"></a>Underground</b></h1>
<p class="p7">Three kilometers underground, in a South African gold mine, scientists discovered [microbe that lives on dirt]</p>
<p class="p8"><br></p>
<p class="p7">There are even microbes that thrive in radioactive environments, like the dangerous interior of the Chernobyl nuclear reactor.</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p13"><img src="Images/Deinococcus radiodurans.png" width="153" height="176" alt="Image"></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p7">Subglacial Lake Whillans is a lake found under more than 800 meters of ice in the West Antarctic. A careful underground bore hole was inserted by a team from Louisiana State University in 2014 [<a href="http://www.nature.com/nature/journal/v512/n7514/full/nature13667.html">http://www.nature.com/nature/journal/v512/n7514/full/nature13667.html</a>]<span class="Apple-converted-space">  </span>and found more than 4,000 different species of bacteria surviving under the ice. The total bacterial count was not that different from the count you'd find in surface lakes on other parts of the planet, a fact that is especially surprising for an environment that hasn't had a ray of light in millions of years. The bacteria instead thrive on iron, sulphur, and nitrogen as energy sources.</p>
<p class="p7"><a href="http://earthsky.org/earth/diverse-microbes-found-deep-beneath-antarctic-ice-sheet">http://earthsky.org/earth/diverse-microbes-found-deep-beneath-antarctic-ice-sheet</a></p>
<p class="p8"><br></p>
<p class="p1"><img src="Images/Pasted Graphic.png" width="580" height="387" alt="Image"></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc37"></a>A crash course in microbiology</b></h1>
<p class="p7">Before we learn how to study your own microbiome, let's review some of the science.</p>
<p class="p8"><br></p>
<p class="p7">All life runs on three chemical building blocks:<span class="Apple-converted-space">  </span>DNA, RNA, and proteins. Each of these is an arbitrarily-long chain of repeating molecules called <span class="s1">nucleotides</span> (DNA or RNA) and <span class="s1">amino acids</span> (proteins). Due to constraints on the way atoms interact, the set of building blocks is fixed. All DNA is composed of only four nucleotides: adenine, thymine, guanine, and cytosine, represented by the letters ATGC. RNA is composed of the same molecules, except that uracile (U) is substituted for thymine.</p>
<p class="p7">Similarly, proteins are constructed with only 20 different amino acids, which can again be represented by a short three-character abbreviation.</p>
<p class="p7">The correspondence between these different proteins and combinations of DNA or RNA is referred to as the genetic code.<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<table cellspacing="0" cellpadding="0" class="t2">
  <tbody>
    <tr>
      <td valign="middle" class="td2">
        <p class="p1"><span class="s2">nonpolar</span></p>
      </td>
      <td valign="middle" class="td3">
        <p class="p1"><span class="s2">polar</span></p>
      </td>
      <td valign="middle" class="td4">
        <p class="p1"><span class="s2">basic</span></p>
      </td>
      <td valign="middle" class="td5">
        <p class="p1"><span class="s2">acidic</span></p>
      </td>
      <td valign="middle" class="td6">
        <p class="p1"><span class="s2">(stop codon)</span></p>
      </td>
    </tr>
  </tbody>
</table>
<p class="p7"><span class="s2"><b>Standard genetic code</b></span></p>
<table cellspacing="0" cellpadding="0" class="t2">
  <tbody>
    <tr>
      <td rowspan="2" valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>1st</b></span></p>
        <p class="p1"><span class="s2"><b>base</b></span></p>
      </td>
      <td colspan="8" valign="middle" class="td8">
        <p class="p1"><span class="s2"><b>2nd base</b></span></p>
      </td>
      <td rowspan="2" valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>3rd</b></span></p>
        <p class="p1"><span class="s2"><b>base</b></span></p>
      </td>
    </tr>
    <tr>
      <td colspan="2" valign="middle" class="td9">
        <p class="p1"><span class="s2"><b>T</b></span></p>
      </td>
      <td colspan="2" valign="middle" class="td10">
        <p class="p1"><span class="s2"><b>C</b></span></p>
      </td>
      <td colspan="2" valign="middle" class="td11">
        <p class="p1"><span class="s2"><b>A</b></span></p>
      </td>
      <td colspan="2" valign="middle" class="td12">
        <p class="p1"><span class="s2"><b>G</b></span></p>
      </td>
    </tr>
    <tr>
      <td rowspan="4" valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>T</b></span></p>
      </td>
      <td valign="middle" class="td13">
        <p class="p1"><span class="s2">TTT</span></p>
      </td>
      <td rowspan="2" valign="middle" class="td14">
        <p class="p14"><span class="s3">(Phe/F) <a href="https://en.wikipedia.org/wiki/Phenylalanine"><span class="s4">Phenylalanine</span></a></span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">TCT</span></p>
      </td>
      <td rowspan="4" valign="middle" class="td16">
        <p class="p1"><span class="s2">(Ser/S) <a href="https://en.wikipedia.org/wiki/Serine"><span class="s5">Serine</span></a></span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">TAT</span></p>
      </td>
      <td rowspan="2" valign="middle" class="td17">
        <p class="p1"><span class="s2">(Tyr/Y) <a href="https://en.wikipedia.org/wiki/Tyrosine"><span class="s5">Tyrosine</span></a></span></p>
      </td>
      <td valign="middle" class="td18">
        <p class="p1"><span class="s2">TGT</span></p>
      </td>
      <td rowspan="2" valign="middle" class="td19">
        <p class="p1"><span class="s2">(Cys/C) <a href="https://en.wikipedia.org/wiki/Cysteine"><span class="s5">Cysteine</span></a></span></p>
      </td>
      <td valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>T</b></span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td13">
        <p class="p1"><span class="s2">TTC</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">TCC</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">TAC</span></p>
      </td>
      <td valign="middle" class="td18">
        <p class="p1"><span class="s2">TGC</span></p>
      </td>
      <td valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>C</b></span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td13">
        <p class="p1"><span class="s2">TTA</span></p>
      </td>
      <td rowspan="6" valign="middle" class="td14">
        <p class="p1"><span class="s2">(Leu/L) <a href="https://en.wikipedia.org/wiki/Leucine"><span class="s5">Leucine</span></a></span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">TCA</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">TAA</span></p>
      </td>
      <td valign="middle" class="td20">
        <p class="p1"><span class="s5"><a href="https://en.wikipedia.org/wiki/Stop_codon">Stop</a></span><span class="s2"> (</span><span class="s4">Ochre</span><span class="s2">)</span></p>
      </td>
      <td valign="middle" class="td18">
        <p class="p1"><span class="s2">TGA</span></p>
      </td>
      <td valign="middle" class="td21">
        <p class="p1"><span class="s5"><a href="https://en.wikipedia.org/wiki/Stop_codon">Stop</a></span><span class="s2"> (</span><span class="s4">Opal</span><span class="s2">)</span></p>
      </td>
      <td valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>A</b></span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td13">
        <p class="p1"><span class="s2">TTG</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">TCG</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">TAG</span></p>
      </td>
      <td valign="middle" class="td20">
        <p class="p1"><span class="s5"><a href="https://en.wikipedia.org/wiki/Stop_codon">Stop</a></span><span class="s2"> (</span><span class="s4">Amber</span><span class="s2">)</span></p>
      </td>
      <td valign="middle" class="td18">
        <p class="p1"><span class="s2">TGG</span></p>
      </td>
      <td valign="middle" class="td22">
        <p class="p1"><span class="s2">(Trp/W) <a href="https://en.wikipedia.org/wiki/Tryptophan"><span class="s5">Tryptophan</span></a>    </span></p>
      </td>
      <td valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>G</b></span></p>
      </td>
    </tr>
    <tr>
      <td rowspan="4" valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>C</b></span></p>
      </td>
      <td valign="middle" class="td13">
        <p class="p1"><span class="s2">CTT</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">CCT</span></p>
      </td>
      <td rowspan="4" valign="middle" class="td23">
        <p class="p1"><span class="s2">(Pro/P) <a href="https://en.wikipedia.org/wiki/Proline"><span class="s5">Proline</span></a></span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">CAT</span></p>
      </td>
      <td rowspan="2" valign="middle" class="td24">
        <p class="p14"><span class="s3">(His/H) <a href="https://en.wikipedia.org/wiki/Histidine"><span class="s4">Histidine</span></a></span></p>
      </td>
      <td valign="middle" class="td18">
        <p class="p1"><span class="s2">CGT</span></p>
      </td>
      <td rowspan="4" valign="middle" class="td25">
        <p class="p1"><span class="s2">(Arg/R) <a href="https://en.wikipedia.org/wiki/Arginine"><span class="s5">Arginine</span></a></span></p>
      </td>
      <td valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>T</b></span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td13">
        <p class="p1"><span class="s2">CTC</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">CCC</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">CAC</span></p>
      </td>
      <td valign="middle" class="td18">
        <p class="p1"><span class="s2">CGC</span></p>
      </td>
      <td valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>C</b></span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td13">
        <p class="p1"><span class="s2">CTA</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">CCA</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">CAA</span></p>
      </td>
      <td rowspan="2" valign="middle" class="td17">
        <p class="p14"><span class="s3">(Gln/Q) <a href="https://en.wikipedia.org/wiki/Glutamine"><span class="s4">Glutamine</span></a></span></p>
      </td>
      <td valign="middle" class="td18">
        <p class="p1"><span class="s2">CGA</span></p>
      </td>
      <td valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>A</b></span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td13">
        <p class="p1"><span class="s2">CTG</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">CCG</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">CAG</span></p>
      </td>
      <td valign="middle" class="td18">
        <p class="p1"><span class="s2">CGG</span></p>
      </td>
      <td valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>G</b></span></p>
      </td>
    </tr>
    <tr>
      <td rowspan="4" valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>A</b></span></p>
      </td>
      <td valign="middle" class="td13">
        <p class="p1"><span class="s2">ATT</span></p>
      </td>
      <td rowspan="3" valign="middle" class="td14">
        <p class="p14"><span class="s3">(Ile/I) <a href="https://en.wikipedia.org/wiki/Isoleucine"><span class="s4">Isoleucine</span></a></span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">ACT</span></p>
      </td>
      <td rowspan="4" valign="middle" class="td16">
        <p class="p1"><span class="s2">(Thr/T) <a href="https://en.wikipedia.org/wiki/Threonine"><span class="s5">Threonine</span></a>        </span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">AAT</span></p>
      </td>
      <td rowspan="2" valign="middle" class="td17">
        <p class="p14"><span class="s3">(Asn/N) <a href="https://en.wikipedia.org/wiki/Asparagine"><span class="s4">Asparagine</span></a></span></p>
      </td>
      <td valign="middle" class="td18">
        <p class="p1"><span class="s2">AGT</span></p>
      </td>
      <td rowspan="2" valign="middle" class="td19">
        <p class="p1"><span class="s2">(Ser/S) <a href="https://en.wikipedia.org/wiki/Serine"><span class="s5">Serine</span></a></span></p>
      </td>
      <td valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>T</b></span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td13">
        <p class="p1"><span class="s2">ATC</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">ACC</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">AAC</span></p>
      </td>
      <td valign="middle" class="td18">
        <p class="p1"><span class="s2">AGC</span></p>
      </td>
      <td valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>C</b></span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td13">
        <p class="p1"><span class="s2">ATA</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">ACA</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">AAA</span></p>
      </td>
      <td rowspan="2" valign="middle" class="td24">
        <p class="p1"><span class="s2">(Lys/K) <a href="https://en.wikipedia.org/wiki/Lysine"><span class="s5">Lysine</span></a></span></p>
      </td>
      <td valign="middle" class="td18">
        <p class="p1"><span class="s2">AGA</span></p>
      </td>
      <td rowspan="2" valign="middle" class="td25">
        <p class="p1"><span class="s2">(Arg/R) <a href="https://en.wikipedia.org/wiki/Arginine"><span class="s5">Arginine</span></a></span></p>
      </td>
      <td valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>A</b></span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td13">
        <p class="p1"><span class="s2">ATG<a href="https://en.wikipedia.org/wiki/DNA_codon_table#endnote_methionineA"><span class="s6">[A]</span></a></span></p>
      </td>
      <td valign="middle" class="td14">
        <p class="p14"><span class="s3">(Met/M) <a href="https://en.wikipedia.org/wiki/Methionine"><span class="s4">Methionine</span></a></span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">ACG</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">AAG</span></p>
      </td>
      <td valign="middle" class="td18">
        <p class="p1"><span class="s2">AGG</span></p>
      </td>
      <td valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>G</b></span></p>
      </td>
    </tr>
    <tr>
      <td rowspan="4" valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>G</b></span></p>
      </td>
      <td valign="middle" class="td13">
        <p class="p1"><span class="s2">GTT</span></p>
      </td>
      <td rowspan="4" valign="middle" class="td14">
        <p class="p1"><span class="s2">(Val/V) <a href="https://en.wikipedia.org/wiki/Valine"><span class="s5">Valine</span></a></span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">GCT</span></p>
      </td>
      <td rowspan="4" valign="middle" class="td23">
        <p class="p1"><span class="s2">(Ala/A) <a href="https://en.wikipedia.org/wiki/Alanine"><span class="s5">Alanine</span></a></span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">GAT</span></p>
      </td>
      <td rowspan="2" valign="middle" class="td26">
        <p class="p14"><span class="s3">(Asp/D) <a href="https://en.wikipedia.org/wiki/Aspartic_acid"><span class="s4">Aspartic acid</span></a></span></p>
      </td>
      <td valign="middle" class="td18">
        <p class="p1"><span class="s2">GGT</span></p>
      </td>
      <td rowspan="4" valign="middle" class="td22">
        <p class="p1"><span class="s2">(Gly/G) <a href="https://en.wikipedia.org/wiki/Glycine"><span class="s5">Glycine</span></a></span></p>
      </td>
      <td valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>T</b></span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td13">
        <p class="p1"><span class="s2">GTC</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">GCC</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">GAC</span></p>
      </td>
      <td valign="middle" class="td18">
        <p class="p1"><span class="s2">GGC</span></p>
      </td>
      <td valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>C</b></span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td13">
        <p class="p1"><span class="s2">GTA</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">GCA</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">GAA</span></p>
      </td>
      <td rowspan="2" valign="middle" class="td26">
        <p class="p14"><span class="s3">(Glu/E) <a href="https://en.wikipedia.org/wiki/Glutamic_acid"><span class="s4">Glutamic acid</span></a></span></p>
      </td>
      <td valign="middle" class="td18">
        <p class="p1"><span class="s2">GGA</span></p>
      </td>
      <td valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>A</b></span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td13">
        <p class="p1"><span class="s2">GTG</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">GCG</span></p>
      </td>
      <td valign="middle" class="td15">
        <p class="p1"><span class="s2">GAG</span></p>
      </td>
      <td valign="middle" class="td18">
        <p class="p1"><span class="s2">GGG</span></p>
      </td>
      <td valign="middle" class="td7">
        <p class="p1"><span class="s2"><b>G</b></span></p>
      </td>
    </tr>
  </tbody>
</table>
<p class="p15"><span class="s6"><a href="https://en.wikipedia.org/wiki/DNA_codon_table#ref_methionineA"><b>A</b></a></span><span class="s2"> The codon ATG both codes for methionine and serves as an initiation site: the first ATG in an <a href="https://en.wikipedia.org/wiki/MRNA"><span class="s5">mRNA</span></a>'s coding region is where translation into protein begins.<a href="https://en.wikipedia.org/wiki/DNA_codon_table#cite_note-pmid19056476-1"><span class="s6">[1]</span></a></span></p>
<p class="p7"><span class="s2"><b>Inverse table (compressed using </b><a href="https://en.wikipedia.org/wiki/Nucleic_acid_notation"><span class="s5"><b>IUPAC notation</b></span></a><b>)</b></span></p>
<table cellspacing="0" cellpadding="0" class="t2">
  <tbody>
    <tr>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Amino acid</b></span></p>
      </td>
      <td valign="middle" class="td28">
        <p class="p1"><span class="s2"><b>Codons</b></span></p>
      </td>
      <td valign="middle" class="td29">
        <p class="p1"><span class="s2"><b>Compressed</b></span></p>
      </td>
      <td rowspan="12" valign="middle" class="td30">
        <p class="p2"><span class="s2"></span><br></p>
      </td>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Amino acid</b></span></p>
      </td>
      <td valign="middle" class="td31">
        <p class="p1"><span class="s2"><b>Codons</b></span></p>
      </td>
      <td valign="middle" class="td29">
        <p class="p1"><span class="s2"><b>Compressed</b></span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Ala/A</b></span></p>
      </td>
      <td valign="middle" class="td32">
        <p class="p1"><span class="s2">GCT, GCC, GCA, GCG</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">GCN</span></p>
      </td>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Leu/L</b></span></p>
      </td>
      <td valign="middle" class="td34">
        <p class="p1"><span class="s2">TTA, TTG, CTT, CTC, CTA, CTG</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">YTR, CTN</span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Arg/R</b></span></p>
      </td>
      <td valign="middle" class="td32">
        <p class="p1"><span class="s2">CGT, CGC, CGA, CGG, AGA, AGG</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">CGN, MGR</span></p>
      </td>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Lys/K</b></span></p>
      </td>
      <td valign="middle" class="td34">
        <p class="p1"><span class="s2">AAA, AAG</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">AAR</span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Asn/N</b></span></p>
      </td>
      <td valign="middle" class="td32">
        <p class="p1"><span class="s2">AAT, AAC</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">AAY</span></p>
      </td>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Met/M</b></span></p>
      </td>
      <td colspan="2" valign="middle" class="td35">
        <p class="p1"><span class="s2">ATG</span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Asp/D</b></span></p>
      </td>
      <td valign="middle" class="td32">
        <p class="p1"><span class="s2">GAT, GAC</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">GAY</span></p>
      </td>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Phe/F</b></span></p>
      </td>
      <td valign="middle" class="td34">
        <p class="p1"><span class="s2">TTT, TTC</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">TTY</span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Cys/C</b></span></p>
      </td>
      <td valign="middle" class="td32">
        <p class="p1"><span class="s2">TGT, TGC</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">TGY</span></p>
      </td>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Pro/P</b></span></p>
      </td>
      <td valign="middle" class="td34">
        <p class="p1"><span class="s2">CCT, CCC, CCA, CCG</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">CCN</span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Gln/Q</b></span></p>
      </td>
      <td valign="middle" class="td32">
        <p class="p1"><span class="s2">CAA, CAG</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">CAR</span></p>
      </td>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Ser/S</b></span></p>
      </td>
      <td valign="middle" class="td34">
        <p class="p1"><span class="s2">TCT, TCC, TCA, TCG, AGT, AGC</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">TCN, AGY</span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Glu/E</b></span></p>
      </td>
      <td valign="middle" class="td32">
        <p class="p1"><span class="s2">GAA, GAG</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">GAR</span></p>
      </td>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Thr/T</b></span></p>
      </td>
      <td valign="middle" class="td34">
        <p class="p1"><span class="s2">ACT, ACC, ACA, ACG</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">ACN</span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Gly/G</b></span></p>
      </td>
      <td valign="middle" class="td32">
        <p class="p1"><span class="s2">GGT, GGC, GGA, GGG</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">GGN</span></p>
      </td>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Trp/W</b></span></p>
      </td>
      <td colspan="2" valign="middle" class="td35">
        <p class="p1"><span class="s2">TGG</span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>His/H</b></span></p>
      </td>
      <td valign="middle" class="td32">
        <p class="p1"><span class="s2">CAT, CAC</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">CAY</span></p>
      </td>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Tyr/Y</b></span></p>
      </td>
      <td valign="middle" class="td34">
        <p class="p1"><span class="s2">TAT, TAC</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">TAY</span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Ile/I</b></span></p>
      </td>
      <td valign="middle" class="td32">
        <p class="p1"><span class="s2">ATT, ATC, ATA</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">ATH</span></p>
      </td>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>Val/V</b></span></p>
      </td>
      <td valign="middle" class="td34">
        <p class="p1"><span class="s2">GTT, GTC, GTA, GTG</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">GTN</span></p>
      </td>
    </tr>
    <tr>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>START</b></span></p>
      </td>
      <td colspan="2" valign="middle" class="td36">
        <p class="p1"><span class="s2">ATG</span></p>
      </td>
      <td valign="middle" class="td27">
        <p class="p1"><span class="s2"><b>STOP</b></span></p>
      </td>
      <td valign="middle" class="td34">
        <p class="p1"><span class="s2">TAA, TGA, TAG</span></p>
      </td>
      <td valign="middle" class="td33">
        <p class="p1"><span class="s2">TAR, TRA</span></p>
      </td>
    </tr>
  </tbody>
</table>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p7">As a programmer looking through all of this, you may immediately be inspired to write your own software version of this. After all, the remarkable consistency between all of these building blocks cries out for manipulation by computer.</p>
<p class="p7">In fact, that's exactly what bioinformaticians do, and numerous software packages have been developed to make it easy to treat these building blocks of life like ordinary computer strings.<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p7">Perhaps the biggest challenge is the volume of data to be handled, which can easily be measured in gigabytes for a simple organism, but can require entire server farms in the case of some real-world biological systems. For that reason, much of bioinformatics is about optimizations to improve the speed of processing a large data set, or to simply the presentation in a way that can reveal the most biologically interesting aspects of a problem without wading in over-complexity.</p>
<p class="p8"><span class="Apple-converted-space"> </span></p>
<p class="p7">One special protein, DNA, can store information</p>
<p class="p8"><br></p>
<p class="p7">The cell</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p7">Start with the cell. All living things are composed of cells, which are tiny self-contained blocks where entropy is kept at bay for as long as possible. Everything in the universe tends over time to fall into disorganized entropy, but cells contain many tricks, honed over billions of years of evolution, to thrive.</p>
<p class="p8"><br></p>
<p class="p7">Some organisms have a single cell, and others are multi-cell, but the key division of life is not between the number of cells but rather in the complexity of its internals. Human cells, like all multicellular life forms, are classified as <span class="s1">eukaryotes</span>, because all of our cells are made of a number of internal features, called <span class="s1">organelles,</span> of which the most prominent is the <span class="s1">nucleus</span>.<span class="Apple-converted-space"> </span></p>
<p class="p7">Bacteria, by contrast, are <span class="s1">prokaryotes</span>. Their cells look much simpler under a microscope: just a cell wall bounding some free-floating objects.<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p7">Part of the definition of life is the ability to reproduce, and cells do this by dividing themselves in half in a process called <span class="s1">mitosis</span>.<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc42"></a>A crash course in botany</b></h1>
<p class="p7">How do you talk about the relationships between various different life forms?</p>
<p class="p8"><br></p>
<p class="p7">A taxon is a simple unit of life. A <span class="s1">homo sapiens</span> is a taxon, but so is a primate. A mammal is a taxon too. It might seem odd in the ordinary biological world to bother using the same term 'taxon' to refer to all of those units, but for bacteria and anything that reproduces asexually, it's an important distinction because often, taxonomists don't agree about whether a group of organisms is part of the same taxon or not.</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p7">Since Carl Linnaeus in the 1700's, the science of taxonomy divides all life into seven major categories: Kingdom, Phylum, Class, Order, Family, Genus, Species (which I was taught in sixth grade to remember by the mnemonic "King Philip Came Over for Girl Scouts").<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p7">Bacteria make up their own kingdom. Just as the animal kingdom includes everything from humans to jellyfish to beetles, the diversity of bacterial life is enormous, a point which can't be emphasized too much. This is true at every rank in the taxonomy. Even two organisms that are the same at a lower rank, like genus, might have radically different affects on the human body, just as a member of the animal genus Canis could be anything from a wolf or coyote to a Chihuahua.<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p7">You cannot mix and match these ranks. If you know something about the number of organisms in one genus, for example, this is meaningful only in comparison to the numbers of another genus. Keep that in mind during our analysis.<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc66"></a>What are they doing?</b></h1>
<p class="p7">Toxoplasma Gondii is a tiny microbe that, for some reason, only likes to reproduce from within the gut of a cat. It can be found in almost all warm-blooded mammals, including humans -- about 30% of us, according to some estimates, and that's after a century of obsession with hygiene that has wiped out countless other tiny inhabitants of the body.</p>
<p class="p7">T. Gondii seem harmless because its hosts appear to show no differences before or after "infection" except in one creature: the mouse. When it finds itself ingested by a mouse, it appears to fade away quietly. MRI scans show large amounts in the gut for a week or two, gradually decreasing until there is apparently nothing. Except: tiny T. Gondii cysts begin to show up in the mouse's brain. Not everywhere, but in just a few strategic places.</p>
<p class="p7">Then something strange happens: the mouse is suddenly attracted to cat urine. And note: its only attraction is to <span class="s1">cat</span> urine, not rabbits, not humans -- only cats. Everything else about the mouse appears normal. It still seems to be afraid of the other things that scare mice: other predators, stressful situations. But put a T. Gondii-affected mouse into a maze with different animal urine in the corners and it will rush to the cat side every time.</p>
<p class="p8"><br></p>
<p class="p7">How is it that T. Gondii is able to be so precise in its effects? It only seems to affect mice, and even then it only apparently makes them attracted to cats. It doesn't apparently cause any other harm to its hosts, so -- somehow -- it must have found a resting place right at the spot in the mouse brain that affects its interest in cats.</p>
<p class="p8"><br></p>
<p class="p7">Although there doesn't seem to be any major negative consequences to humans hosting T. Gondii, some scientists aren't sure. One man in particular, Jaroslav Flegr, an evolutionary biologist at Charles University in Prague, thinks he has evidence that women who carry it might be more trusting than those who don't.<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p7">T.Gondii isn't the only microbe known to affect the behavior of its host. A more common example is the rabies virus, which upon infection somehow causes a mammal to be more agitated, more likely to strike out -- or bite -- other humans, thereby spreading itself.</p>
<p class="p8"><br></p>
<p class="p7">Or syphilus, the disease spread by the bacterium [syphilius] and causes its host to go insane. The microbe is somehow able to infect the mind of the victim.</p>
<p class="p7">In general, sexually transmitted diseases are especially likely to have behavioral consequences. A sexual disease that produces symptoms is unlikely to spread, yet it still requires contact with a new victim. Perhaps the ideal vector is a behavioral change, making the host more likely to come in contact with a new host.</p>
<p class="p8"><br></p>
<p class="p7">Think about this too much, and you'll end up with the obvious question: what other weird microbes are infecting us right now? Can we explain some of our own behaviors this way? Is there a human equivalent of these infections, driving us to do things we "ordinarily" wouldn't do? And maybe these microbes are so ubiquitous, teeming all over us and in our brains, maybe there's no way to even know what "ordinary" or "normal" human behavior is.</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc70"></a>Studying the Human Microbiome</b></h1>
<p class="p7">Studying the Human Microbiome</p>
<p class="p8"><br></p>
<p class="p7">During the late 1990s, biologists around the world were racing to complete the first sequence of the human genome, a first draft of which was available in June 2000, and the project was declared complete in 2003. With this sequence available, scientists now had the first rough map of all the genes that make up a human being, and they were hopeful that this would be quickly followed by new discoveries linking genes to health.<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p7">The Human Microbiome Project was launched in 2008 by the National Institutes of Health with a budget of $115M.</p>
<p class="p8"><br></p>
<hr>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 24.0px Georgia"><b><a id="doc38"></a>Chapter Two: The Technology</b></h1>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 24.0px Georgia; min-height: 27.0px"><br></h1>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc64"></a>Studying the Microbiome</b></h1>
<p class="p7">Two major problems had to be solved before scientists could begin seriously studying the microbiome.</p>
<p class="p7">First, the various bacteria inside us had to be identified. Since the invention of the microscope, the process was straightforward: collect enough of the organisms of interest and then look at them, either as a whole while cultured in a lab, or individually under the microsoft.</p>
<p class="p8"><br></p>
<p class="p7">Computer scientists have an easier time than biologists because operations are so fast you can afford to do them over and over. In biology, individual operations are much slower, but you can run them in parallel. That goop in the petri dish is often made of trillions of organisms, each one consuming, producing, and multiplying individually. Even with the very low error rates in ordinary cell division, you have so many organisms that you will regularly get a few mutations, and if the experiment is well-controlled, you will be able to see those mutations.</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc40"></a>The old way: culturing</b></h1>
<p class="p7">How do you tell which microbe is which?</p>
<p class="p8"><br></p>
<p class="p7">Until recently, the old fashioned way was the only way. You get some of the bacterium of interest and then grow more of it. In other words, a glorified version of farming.</p>
<p class="p8"><br></p>
<p class="p7">How do you farm a microbe? Basically, you put it in a petri dish along with something that it likes to eat, and you wait for it to grow. What does it like to eat? Well, that's the "art". If you find the right substance and just the right environment, it might grow. Then again, it might not. Much of this is trial and error.</p>
<p class="p7">A bigger problem with culturing is that some organisms need other organisms in order to survive. You may be simply unable to grow the one of interest unless you have a whole colony of other, unrelated but symbiont organisms nearby to provide it something it needs. In other words, it may simply be impossible to grow a bunch of one particular organism in isolation.</p>
<p class="p8"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc39"></a>16s rRNA</b></h1>
<p class="p7">The 16S shortcut</p>
<p class="p8"><br></p>
<p class="p7">A living organism can contain somewhere between thousands and hundreds of billions of DNA base pairs. There is some commonality between related organisms -- humans and chimpanzees, for example, share upwards of 90% of their DNA -- but in general it's hard to use the DNA strand itself to measure the relatedness between two organisms. Understanding the reason for this may help you understand why there is a clever shortcut.</p>
<p class="p7">You might think you can measure the relatedness of two organisms by looking at all the DNA in each one, and computing the percentage that each shares in common. This would work, but sequencing all those billions of DNA bases takes a lot of time and money, and it would be impractical in a case like the microbiome where you may need to do this for millions of individual organisms.</p>
<p class="p7">A service like 23andme is able to cheaply compare individuals of the same species (i.e. Humans) because the generic human genome is already well-mapped and we know that of the 3 billion base pairs, only about 3 <span class="s1">million</span> (the SNPs or single-nucleotide polymorphisms) are different between individuals. When you give your spit sample to 23andme, they give you back a subset of your SNPs, only those that have been studied enough to be interesting. SNPs are easy and cheap to find using a "gene-chip", a special semiconductor-like device that can quickly look at 1 million or more pre-determined spots on your DNA. But this is only possible because the map itself already exists, thanks to multi-year effort of the Human Genome Project that finished in the early 2000s. There are no gene chips (yet) for bacteria, and certainly not for all the millions of types in your body. And even if there <span class="s1">were</span> such chips, bacteria are notorious at adapting and changing to their surrounding environment, exchanging genes with one another, that it just wouldn't be practical to identify enough constant genes to make it worthwhile.<span class="Apple-converted-space"> </span></p>
<p class="p7">Fortunately, some parts of an organisms DNA change more often than others. The SNPs are reliably different between any two individuals of the same species, but some of parts of DNA are so important that they stay virtually identical even across entire families of organisms. Something like skin pigmentation, for example, can change without having much effect at all on survival, but remember that DNA describes absolutely <span class="s1">everything</span> about the organism, including the workings of very low-level cell process. Not just the skin pigmentation, or even how skin cells grow, but much more fundamental: how a cell divides, for example, or even how to use the oxygen a cell needs for survival.</p>
<p class="p7">Among the most fundamental of all processes is the what happens in every cell's ribosome, a special enzyme that knows how to copy DNA. The ribosome is such a fundamental part of every living organism that very little about the ribosome has changed in a billion years of evolution. Humans and corn plants actually share quite a bit of the ribosome; both are prokaryotes, for one thing, so many of our cellular processes work the same.</p>
<p class="p7">But bacteria go back even further than humans and corn plants, enough so that the differences aren't so subtle anymore. In fact, the differences are big enough that you don't even need to do gene sequencing to peek at one of the most<span class="Apple-converted-space"> </span></p>
<p class="p7">If you cut up bacterial cells ("lyse" them, to use the technical term) to pull out their DNA, and you throw it into a centrifuge turning at a carefully controlled speed, some the "goo" from the cells will rise further than others because some parts of the cell are heavier. One specific part of the cell, corresponding to a section of the cell's ribosomes, will rise to a centrifuge level referred to as "16S". Precisely skimming the goo at that spot will give you a collection of RNA from just one part of the ribosome of bacteria. Other cell types that may happen to be in the sample will not show up at that spot on the centrifuge. That's the shortcut.</p>
<p class="p7">Once you have a bunch of that 16S ribosomal RNA, you know that you have bacterial RNA and <span class="s1">that's</span> what the Illumina NextSeq 500 will sequence. This represents a huge subset, a nearly trivial percentage of all the DNA in the overall sample, but combined with one more shortcut, it's enough to accurately identify the sample.</p>
<p class="p7">The remaining shortcut is possible thanks to years of research of sequencing the genes in bacteria. Scientists in labs around the world have been faithfully digging up samples of bacteria, and performing whole-gene sequencing on what they find. These sequences are kept in a<span class="Apple-converted-space">  </span>large public database run by the National Institutes of Health, and if you have a snippet of DNA -- even a small snippet -- you can do a string compare against all the genes that have already been analyzed. When you find a match, you can be pretty sure that your sample contains this type of bacteria.</p>
<p class="p7">It's this two-step combination, 16S "skimming" and a database lookup, that makes it cost effective to study the millions of organisms in your microbiome. You don't have to do a complete gene sequence on every single bacterium; just trust that the tiny subset of RNA in the 16S region is enough to uniquely match something already in the bacterial database.</p>
<p class="p8"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc43"></a>Gene sequencing</b></h1>
<p class="p7">Gene sequencing</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc44"></a>Bioinformatics</b></h1>
<p class="p7">Once you have the data computerized, the analysis tools leave the wet world of biology and enter the software world of computing. This marriage is called bioinformatics.</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<hr>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 24.0px Georgia"><b><a id="doc45"></a>Chapter Three: How to Analyze Your Microbiome</b></h1>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 24.0px Georgia; min-height: 27.0px"><br></h1>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc47"></a>How to get your data</b></h1>
<p class="p7">There are several ways for an individual to get access to the powerful sequencers that make 16S rRNA analysis possible.<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p7">That said, just as I mentioned 23andme as a way to find information about your DNA, there are a few ways you can get information about your microbiome.<span class="Apple-converted-space">  </span>One way is through a doctor, or other health care provider who can give you a clinical test, just like you might do a blood draw as part of an annual physical. If you have a medical condition, the tests are likely covered by insurance, but tend to be pretty expensive (several hundred dollars or more).<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p7">For "normal" people, just curious about the science like I am, there are two other ways to find out your microbiome:</p>
<p class="p8"><br></p>
<p class="p7"><span class="Apple-converted-space"> </span>The first is called American Gut Project, associated with a lab at the University of California San Diego and others.<span class="Apple-converted-space">  </span>For a $100 donation, they'll send you a sampling kit which you send back to get a breakdown of your gut bacteria. American Gut boasts an impressive number of practicing research scientists, including several of the key contributors to the Human Microbiome<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p13"><img src="Images/American Gut.png" width="245" height="111" alt="Image"></p>
<p class="p7">I recommend the San Francisco company uBiome.</p>
<p class="p8"><br></p>
<p class="p7"><b>History of uBiome</b></p>
<p class="p7">The company was founded in late 2012 by Jessica Richman and Zach Apte, who met while doing graduate studies at Oxford. Jessica, with an undergraduate degree from Stanford, was studying public policy, using computers to understand how new ideas go viral. Zach was a biologist studying the microbiome, disappointed at how long it took to collect the vast number of samples required to do the science. The problem, they realized, was not lack of interest from the general public. There was no shortage of people interested in the microbiome and its affect on health, many of whom would happily contribute samples if they could help the science. They suspected that many of these "citizen scientists" were so eager to help that they might even <span class="s1">pay</span> for the privilege of participating in an experiment, particularly if they could receive personalized results of their own tests.</p>
<p class="p7">In November 2012, they launched their first crowd funding campaign on the site Indiegogo, hoping to raise $100,000 to start the business. They started by selling "perks", $89 for a gut kit, $399 for a five-sample kit, offering to run the samples through the sequencing machine that Zach Apte was using at the University of California San Francisco biotech startup garage. To their surprise, the campaign raised more than $350,000, and the company was off and running.</p>
<p class="p7">Since uBiome focused from the beginning on crowd-sourced open science, their kits were designed to be as easy to use as possible. With glossy-printed, step-by-step directions and an attractive kit with well-labeled test tubes, they wanted to let ordinary people take samples and submit them with as little fuss as possible.</p>
<p class="p7">At the lab, uBiome also focused on automation. They built two special collection robots that could open the test tubes with the ultimate goal that they'd be able to submit to the sequencing machine with barely any human intervention.</p>
<p class="p13"><img src="Images/uBiomeRobot20130917001011-robot.jpg" width="504" height="503" alt="Image"></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p7">The strong start with the Indiegogo campaign brought them enough money to start the business, but more importantly it attracted the attention of other investors who soon contributed $1.5M to the new company. Eventually the uBiome staff signed on to the prestigious Y-Combinator, the startup incubator that has given birth to many famous names including Airbnb, Reddit, Dropbox and others, and landed an additional $3M investment from the prestigious venture capital firm Andreessen Horowitz.</p>
<p class="p7">With the new investment capital, uBiome was able to open a new headquarters and buy their own $300,000 NextSeq500 gene sequencing machine and more automation to help them process the samples more quickly.</p>
<p class="p8"><br></p>
<p class="p7">The most popular gene sequencer is the NextSeq500, from San Diego-based Illumina.</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc46"></a>The basics: web tools</b></h1>
<p class="p7">After uBiome has notified you of your results, you can go to <a href="http://app.ubiome.com">their web site</a> to learn more.</p>
<p class="p8"><br></p>
<p class="p7">The first thing you'll notice is a pretty chart like this one:</p>
<p class="p13"><img src="Images/uBiomeGutChart.jpg" width="600" height="389" alt="Image"></p>
<p class="p8"><br></p>
<p class="p7">This shows the overall breakdown of the bacteria they found in your gut. In this chart, the bacteria are lumped by phyla --the highest, broadest level of classification. If you're like most Americans, the vast majority of your sample will be one of two phyla: <span class="s1">bacteroidetes</span> or <span class="s1">firmicutes</span>.<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p13"><img src="Images/uBiomeGutSample.jpg" width="580" height="293" alt="Image"></p>
<p class="p16"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc56"></a>Analyze with Excel</b></h1>
<p class="p7">The Illumina machines spit off huge amounts of data (FASTQ files can be hundreds of thousands of lines long), so to be useful you'll need a summary. UBiome provides a much more concise version called a "raw taxonomy" file, which is generally only a few hundred lines showing just the organisms they think they found in the sample.</p>
<p class="p8"><br></p>
<p class="p7"><b>What is JSON</b></p>
<p class="p7">UBiome's taxonomy files come in a simple structured text format called JSON (JavaScript Object Notation), commonly used across the web. Although it's not as convenient for people as the rest of the uBiome web site, programmers refer to JSON as "human readable", because if you squint enough you can sorta tell what it means without a computer program. Here's a sample:</p>
<p class="p8"><br></p>
<p class="p17">{</p>
<p class="p17"><span class="Apple-converted-space">  </span>"ubiome_bacteriacounts": [{</p>
<p class="p17"><span class="Apple-converted-space">    </span>"taxon": "2",</p>
<p class="p17"><span class="Apple-converted-space">    </span>"parent": "131567",</p>
<p class="p17"><span class="Apple-converted-space">    </span>"count": "22691",</p>
<p class="p17"><span class="Apple-converted-space">    </span>"count_norm": "1000000",</p>
<p class="p17"><span class="Apple-converted-space">    </span>"avg": null,</p>
<p class="p17"><span class="Apple-converted-space">    </span>"tax_name": "Bacteria",</p>
<p class="p17"><span class="Apple-converted-space">    </span>"tax_rank": "superkingdom",</p>
<p class="p17"><span class="Apple-converted-space">    </span>"tax_color": null</p>
<p class="p17"><span class="Apple-converted-space">  </span>}, {</p>
<p class="p17"><span class="Apple-converted-space">    </span>"taxon": "1239",</p>
<p class="p17"><span class="Apple-converted-space">    </span>"parent": "2",</p>
<p class="p17"><span class="Apple-converted-space">    </span>"count": "15414",</p>
<p class="p17"><span class="Apple-converted-space">    </span>"count_norm": "679300",</p>
<p class="p17"><span class="Apple-converted-space">    </span>"avg": null,</p>
<p class="p17"><span class="Apple-converted-space">    </span>"tax_name": "Firmicutes",</p>
<p class="p17"><span class="Apple-converted-space">    </span>"tax_rank": "phylum",</p>
<p class="p17"><span class="Apple-converted-space">    </span>"tax_color": "5E6591"</p>
<p class="p17"><span class="Apple-converted-space">  </span>}</p>
<p class="p8"><br></p>
<p class="p7">JSON is just structured data. That's it.<span class="Apple-converted-space">  </span>The data is organized according to <span class="s1">keys</span> and <span class="s1">values</span>. The <span class="s1">keys</span> are unique identifiers and the values are their values. The whole file looks that way. The only requirement of a JSON file is that it be consistent with this pattern, because after all it will need to be read by a computer eventually.</p>
<p class="p7">The uBiome Taxonomy JSON files identify the key ```uBiome_bacteriacounts``` which is mapped to a bunch of fields, each representing a particular organism found in your sample. Look at the figure above to get the basic idea.</p>
<p class="p8"><br></p>
<p class="p7">Because uBiome JSON files are well-structured, it's easy to process them with other software, including Excel. That's our next step: bring this data into Excel.</p>
<p class="p8"><br></p>
<p class="p7"><b>Converting uBiome taxonomy JSON to Excel</b></p>
<p class="p8"><br></p>
<p class="p7">If you just have one or two files, it's easy to bring uBiome taxonomy data into Excel. Go to the uBiome apps page and click on the section labeled "Download taxonomy".</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p13"><img src="Images/uBiomeScreenDownloadData.png" width="290" height="325" alt="Image"></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p7">You'll see a page of JSON representing what uBiome found in your sample.</p>
<p class="p8"><br></p>
<p class="p13"><img src="Images/uBiomeScreenShotTaxonomyJSON.png" width="471" height="225" alt="Image"></p>
<p class="p8"><br></p>
<p class="p7">On a desktop computer, if you select-all (press control-A or command-A) then you can copy this data to the clipboard to prepare to paste it to another web site.</p>
<p class="p8"><br></p>
<p class="p7">There are many web sites that will convert JSON to a format that will work in Excel. Google "Json to Excel" or "Json to CSV" and you'll find one. Just paste your data into one of those sites to convert it for either Excel's native (XLS or XLSX) format, or the more universal CSV format, readable by Excel. Here are a few of the sites I've tested:</p>
<p class="p7"><a href="http://www.convertcsv.com/json-to-csv.htm">http://www.convertcsv.com/json-to-csv.htm</a></p>
<p class="p7"><a href="http://www.json-xls.com/json2xls">http://www.json-xls.com/json2xls</a></p>
<p class="p8"><br></p>
<p class="p7">Once the data is in Excel, you can work with it just as you would with any Excel data sheet. Here's how one of my recent uBiome taxonomy files looks in Mac Excel:</p>
<p class="p8"><br></p>
<p class="p13"><img src="Images/ExcelScreenOverview.png" width="505" height="311" alt="Image"></p>
<p class="p8"><br></p>
<p class="p7">Yours may look slightly different. Sometimes uBiome changes the labels on the taxonomy slightly, and the ordering of the columns may be different depending on how the JSON was converted, but none of that matters. The key is that you are now able to work with it in a full-blown spreadsheet.</p>
<p class="p8"><br></p>
<p class="p7"><b>The uBiome data tags</b></p>
<p class="p7">Let's review the meaning of each of the columns. The uBiome JSON taxonomy data includes the following fields:</p>
<p class="p8"><br></p>
<p class="p7">The uBiome JSON taxonomy data includes the following fields:</p>
<p class="p8"><br></p>
<p class="p7"><b>count</b>: an absolute measure of number of organisms found in the sample. Without knowing the size of the sample, or how many times the DNA inside was processed through PCR amplification, this number doesn't mean much except in relation to other counts at the same taxonomical rank.</p>
<p class="p8"><br></p>
<p class="p7"><b>count_norm:</b> a "normalized" version of the count, created by simply dividing the count on a given row by the count found on the row with tax_name = Bacteria and then multiplying by one million. It's easier to just think of it as parts per million: each unit is 1 / 10,000th of a percent. For example, if you see a row with count_norm = 500,000, you can just think of that as 50% of the sample.</p>
<p class="p8"><br></p>
<p class="p7"><b>tax_name</b>: this is the classification of the organism based on the level of its taxonomy. If you were looking at a human being, for example, you would see homo sapiens if you selected tax_rank = species, but you'd see mammalia if you selected tax_rank = class.</p>
<p class="p8"><br></p>
<p class="p7"><b>tax_rank</b>: tells the level of the taxonomy. In daily conversation about animals or plants, we usually refer to the species (e.g. homo sapiens), but sometimes it's more useful to talk about bigger groupings of related organisms. For example, humans are members of the class mammalia, along with tigers and horses. If this spreadsheet were counting organisms at the level of class mammalia, the count_norm would almost certainly be bigger than the count_norm for humans alone, unless humans were the only type of mammal found in the sample. <span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p7"><b>taxon and parent:</b> these help identify the ranking in a more precise way by pointing out which tax_ranks are subsets of which. For example, Bacteroidia above has a parent = 976, meaning that it is a subset of the taxon 976, Bacteroidetes. When you follow the various taxons and parents up the chain, you'll see they all end in the superkingdom Bacteria, which has a taxon of 2. The values for these numbers, incidentally, are taxonomical numbers from the curated database at NCBI, the national bioinformatics center run by the U.S. government. Enter the number into the taxonomy browser at the NCBI Taxonomy Browser and you can learn as much as you want about that organism.</p>
<p class="p8"><br></p>
<p class="p7"><b>tax_color </b>doesn't matter for this anaysis, but uBiome software uses this to colorize their pretty graphs to make them more readable.</p>
<p class="p8"><br></p>
<p class="p7">Of these fields, the most important ones are <b>tax_name</b>, <b>tax_rank,</b> and <b>count_norm</b>.<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc72"></a>Comparing samples</b></h1>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia; min-height: 19.0px"><b></b><br></h1>
<p class="p7">The uBiome JSON taxonomy file is extremely helpful at giving you a short summary of what you care about most in a sample: the organisms within it and their relative amounts. But even with these annotations, there's a limit to what you can learn in a single sample. To be able to start hacking your microbiome, you will need to learn how to compare multiple samples.</p>
<p class="p7">There are two main things you'll want to measure when comparing two samples:</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p7"><b>Uniqueness</b>: which organisms are found in only one sample and not the other? If you are comparing two samples from the same individual (e.g. Yourself), then uniqueness is another way of talking about either <span class="s1">extinctions</span> (when a taxon has disappeared entirely in a later sample) and <span class="s1">appearances</span> (when a new taxon magically shows up). In other words, you care a lot about the order in which the samples were taken.</p>
<p class="p7">On the other hand, if you're comparing two separate individuals, then the ordering of the samples doesn't matter. Uniqueness just tells which organisms are unique to a specific person.</p>
<p class="p8"><br></p>
<p class="p7"><b>Relative abundance</b>: which person has more of which organism? There are two senses in which we care about the relative frequency of an organism's occurrence in a sample. You might care, among all the taxons found, which ones are more highly represented in <span class="s1">absolute</span> amounts?<span class="Apple-converted-space">  </span>Or you might care on a relative basis. To help understand why this matters, it may help to think of the following specific example:</p>
<p class="p7">In many Americans the phylum <span class="s1">Firmicutes</span> makes up a majority of the sample, often reaching 60% or more. You might find two people, one who has 60% <span class="s1">Firmicutes</span> and the other with 30% -- only half as much. The relative difference is 2x and the absolute difference is 30 percentage points.</p>
<p class="p7">On the other hand, you might find <span class="s1">Bifidobacterium</span> makes up 10% of the first sample and 5% of the second. In this case, the relative difference is 2x -- just like the <span class="s1">Firmicutes</span> case -- but the <span class="s1">absolute</span> difference is only 5 percentage points, much less than in the first example.<span class="Apple-converted-space"> </span></p>
<p class="p7">The built-in uBiome web tools look for <span class="s1">relative</span> differences. A taxon might be a minuscule component of both samples, but when you start from a small base, even a few additional organisms can make up a big percentage change. In the extreme case, a sample with a <b>count_norm</b> of 100 compared to another sample with a <b>count_norm</b> of 300 will show a 3X increase -- very high in aBiome terms, even though the absolute difference is only 300 - 100 = 200 organisms.</p>
<p class="p7">Compare that with a an organism with a <b>count_norm </b>of 50,000 in one sample and 51,000 in another sample. UBiome's algorithms will treat these as a virtual tie, even though the absolute number of organisms in the second sample outnumber the first by 1,000 organisms.</p>
<p class="p8"><br></p>
<p class="p7">Which method of measuring is better? It depend on what you care more about.<span class="Apple-converted-space">  </span>Some species need have only a tiny representation in a sample to make a big difference in health outcomes.<span class="Apple-converted-space"> </span></p>
<p class="p7">I prefer to use both methods of measurement: absolute number changes matter for the most popular taxons, and relative amounts matter more for the less-popular ones.</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<hr>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 24.0px Georgia"><b><a id="doc51"></a>Chapter Four: Organisms to Watch</b></h1>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 24.0px Georgia; min-height: 27.0px"><br></h1>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc52"></a>Good and Bad</b></h1>
<p class="p7">As tempting as it is to categorize bacteria into groups like "good" and "bad", you should avoid that mistake. Very few organisms are always and everywhere either beneficial or harmful. Everything depends on the circumstances.</p>
<p class="p7">For example, from [Blaser p119]</p>
<p class="p8"><br></p>
<p class="p7">"viridans" streptococci<span class="Apple-converted-space">  </span>"live peacefully in everyone's mouth...a leading cause of heart valve infections...normal residents of the mouth, only occasionally entering the bloodstream and landing on a previously damaged valve....if we mix harmless Viridans with pathogenic Group A strep, viridans always win out. They knock back the strep."</p>
<p class="p8"><br></p>
<p class="p7">Other examples:</p>
<p class="p8"><br></p>
<p class="p7">H.Pylori</p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc75"></a>Gut Guardians</b></h1>
<p class="p7"><b>The Gut Guardians</b></p>
<p class="p7">Grace Liu is a PharmD from the Bay Area who has been fascinated by the gut microbiome for many years. She focuses on the following organisms she calls the "gut guardians", the keystone species that, when properly groomed can<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p7"><b>The importance of butyrate</b></p>
<p class="p8"><br></p>
<p class="p7">The cells lining your intestines will die without proper nourishment, and a key food they expect is butyrate, a type of short-chain fatty acid. Fortunately, your colon is covered with bacteria that can ferment plant fiber, producing butyrate as a by-product.</p>
<p class="p7">From Wikipedia:</p>
<p class="p17">Butyrates are important as food for cells lining the mammalian colon (colonocytes). Without butyrates for energy, colon cells undergo autophagy (self digestion) and die.[1] Short-chain fatty acids, which include butyrate, are produced by beneficial colonic bacteria (probiotics) that feed on, or ferment prebiotics, which are plant products that contain adequate amounts of dietary fiber. These short-chain fatty acids benefit the colonocyte by increasing energy production,and cell proliferation and may protect against colon cancer.[2]</p>
<p class="p19"><br></p>
<p class="p17">Butyrate is a major metabolite in colonic lumen arising from bacterial fermentation of dietary fiber and has been shown to be a critical mediator of the colonic inflammatory response.<span class="Apple-converted-space"> </span></p>
<p class="p7"><b>Faecalibacterium prausnitzii</b></p>
<p class="p8"><br></p>
<p class="p7"><a href="http://www.sciencedirect.com/science/article/pii/S1369527413000775">http://www.sciencedirect.com/science/article/pii/S1369527413000775</a></p>
<p class="p20"><span class="s4">Faecalibacterium prausnitzii</span><span class="s2"> is the most abundant bacterium in the human intestinal microbiota of healthy adults, representing more than 5% of the total bacterial population. Over the past five years, an increasing number of studies have clearly described the importance of this highly metabolically active commensal bacterium as a component of the healthy human microbiota. Changes in the abundance of </span><span class="s4">F. prausnitzii</span><span class="s2"> have been linked to dysbiosis in several human disorders.<span class="Apple-converted-space"> </span></span></p>
<p class="p21"><br></p>
<p class="p21"><br></p>
<p class="p22"><b>Roseburia</b></p>
<p class="p22">This genus, under the phylum <span class="s1">firmicutes</span>, is known to be associated with health.</p>
<p class="p21"><br></p>
<p class="p21"><br></p>
<p class="p22">Christensenellaceae</p>
<p class="p22">Christensenella</p>
<p class="p22">Akkermansia</p>
<p class="p22">Bifidobacteria longum</p>
<p class="p22">Bifidobacterium</p>
<p class="p21"><br></p>
<p class="p22">B. Longum as % of total Bifido</p>
<p class="p22">Bacteroidetes/Firmicutes ratio</p>
<p class="p21"><br></p>
<p class="p21"><br></p>
<hr>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 24.0px Georgia"><b><a id="doc49"></a>Chapter Five: Next Steps</b></h1>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; text-align: center; font: 24.0px Georgia; min-height: 27.0px"><br></h1>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc50"></a>Fermented food</b></h1>
<p class="p7">Once you've developed an appreciation for the importance of the microbiome, you'll want to become more of a farmer, growing your own symbionts.</p>
<p class="p8"><br></p>
<p class="p7">People have been fermenting food since the distant past, making everything from beer to cheese. Fermentation is an ideal way to preserve food beyond the date at which it is practical to eat.</p>
<p class="p8"><br></p>
<p class="p7">Fermenting your own foods is surprisingly easy.</p>
<p class="p8"><br></p>
<p class="p7"><b>Yogurt</b></p>
<p class="p7">One of the easiest ferments is home-made yogurt. Pour a few cups of whole milk into a saucepan and heat to not-quite-boiling. You'll need to get it above 160 degrees, the temperature that kills bacteria in milk, but if it goes all the way to boiling the milk will taste scalded. Even that's not the end of the world, but you may as well try to keep it under that. Use a kitchen thermometer the first few times if you like, but soon you'll be able to tell intuitively because you'll see steam rising from the surface of the milk, but no boiling bubbles.</p>
<p class="p7">After the milk has been heated, remove and pour it into a glass bowl. Let it sit until it cools to about 110 degrees. Here, a kitchen thermometer is more useful, especially the first time because the exact temperature is more important. Warmer than 110 degrees and the <span class="s1">lactobacillus</span> yogurt bacteria will die; too much lower and they won't grow. Once you have the hang of it, you'll be able to tell based on the touch: it'll feel unpleasantly warmer than your hand (which should be 98 degrees) but not too warm to touch.</p>
<p class="p7">Once the milk hits that rough temperature zone, pour in a tiny amount of yogurt from another source.<span class="Apple-converted-space">  </span>This is called the "starter" yogurt and the best is to use a bit of leftover from your previous batch, but for your first attempt you can use any commercially-purchased yogurt. Sweetened, flavored, non-fat -- any kind is okay as long as it has live cultures in it.</p>
<p class="p7">Be sure not to use too much yogurt. Definitely no more than a tablespoon, and perhaps even just a teaspoon or two. Because of the small amounts, you'll be tempted to think you need more, but in this case more is definitely not better. Too much starter will suffocate the mixture. Whenever I've had an unsuccessful batch it's almost always due to having too much starter.</p>
<p class="p7">Mix the starter thoroughly into the lukewarm milk and then cover it up and put it someplace warm. The key is to keep the mixture close to that magical 110 degree point. Some people put it overnight in an oven set to the lowest temperature, but if you don't like running the oven overnight you can try wrapping it tightly with something that will retain the heat through the night. If have a high-quality thermos-style cooler, you can set it inside there, and some people cover it with thick towels and set it in the warmest part of the house. Another option is to use a heating pad, set to the lowest setting and placed underneath the bowl. The important thing is to keep the mixture at the warm temperatures that the bacteria need in order to breed.</p>
<p class="p7">Set in a quiet place for about 8-10 hours. Overnight is ideal. When you open the bowl again, you'll find the mixture no longer resembles milk, but has been transformed into a much thicker consistency -- a yogurt. It will taste sour, and if you're not accustomed to plain, unsweetened yogurt you may even think that is has spoiled. Nope, that's just how pure, real yogurt is meant to taste.</p>
<p class="p8"><br></p>
<p class="p7"><b>Pickling</b></p>
<p class="p7">The next step after yogurt-making is also easy. It takes a little longer, from one to several weeks, but lets you expand your fermentation skills to many more foods. The most basic one is cabbage, to make your own sauerkraut.</p>
<p class="p7">Think of fermentation as a type of farming, only instead of large plants that you see, you're farming with invisible microbes. In both cases the object is to start with a small seed that grows into a full crop. For sauerkraut, the microbes are naturally occurring on the leaves of plants and vegetables. You just need to put them into the right environment so the microbes can grow.<span class="Apple-converted-space"> </span></p>
<p class="p7">Start with a small head of cabbage and a grater. If you don't have a grater, then just chop it into small pieces. Cabbage is already covered with tiny microbes, but the interesting ones will not reproduce in the presence of oxygen; you need to get them out of the air. The best way to do that is to submerge them in a liquid -- in this case we'll use the naturally occurring water inside the cabbage leaves themselves. The purpose of grating or chopping is simply to let the water escape from the leaves until it covers the entire mixture.</p>
<p class="p7">You'll need some type of container to submerge the leaves. Best is something made of glass or ceramic, though any container is fine as long as it doesn't react with the acids that will be produced by the microbes. A standard mason jar works, for example, or even a cleaned-out jar of pasta sauce. Make sure you have a lid for the container, and make sure the lid doesn't close too tightly. If you use something with a screw-top lid (like a pasta sauce jar) just pop a few tiny holes in the lid. The microbes will be producing carbon dioxide as they ferment, and you'll need a way for the gases to leave or the container will explode.</p>
<p class="p7">Begin packing the cabbage leaves into the container as tightly as possible. While doing this, most people add some salt. The salt serves two purposes; it adds flavor, and it helps draw more water out of the plant. Just a small amount of salt is enough: about what you would use if you were seasoning it for flavor: a few shakes on each cabbage leaf as you peel it is enough, or maybe a total of a tablespoon or two for a small head of cabbage. Continue to pile more leaves into the container until it is full.</p>
<p class="p7">If you're doing everything correctly, there will be a nice layer of liquid that completely covers the leaves in the container. Remember that the microbes can't reproduce if there is air, so covering with liquid is essential. If you see any green leaves poking out of the liquid, pack them down further. It may help to find an insert of some kind to place directly over the leaves, perhaps anthe lid to a smaller sized jar, one that fits inside the container and lets you squish the leaves even tighter. The key is to keep everything submerged in liquid.</p>
<p class="p7">Put as many cabbage leaves as you can into the container and close the lid, but be careful: the bacteria in the jar will soon start to produce carbon dioxide gas, which will put pressure on the container. That's the reason for the air holes you poked in the lid beforehand. You may want to cover the entire jar with a loose-fitting piece of cellophane, or an old cloth, as a precaution in case the aroma begins to attract insects.</p>
<p class="p7">That's it! Leave the fermenting jar in a cool, undisturbed place and then check on it for the first few days. Make sure the liquid has entirely submerged the cabbage leaves. Within a day or two you'll also notice some air bubbles in the liquid -- the sign that the fermentation has begun to work. Now, just let the jar rest for another several days or more. The longer it sits, the more acidic it will become. If you like a sour taste, let it sit for a week or two, maybe even up to a month or more. If you want a more subtle flavor, take it out in a few days. Feel free to sniff the jar every day or two to decide when you think it's ready.</p>
<p class="p7">When finished, it will look and smell like sauerkraut. If some of the leaves were not entirely submerged in liquid, it's possible that a layer of moldy fungus has appeared, unsightly but not a problem. Just scrape off whatever doesn't look like sauerkraut and eat the rest.</p>
<p class="p8"><br></p>
<p class="p7"><b>What's happening in sauerkraut</b></p>
<p class="p7">The transformation from cabbage leaves to sauerkraut happens thanks to a complex interaction of bacteria starting with some members of the Leuconostoc species that occur naturally on the cabbage leaves. These are anaerobic organisms -- they can't live in oxygen. When submerged in the cabbage liquid, however, they thrive, consuming all the sugars they can find and converting them to lactic acid, giving off carbon dioxide as a byproduct. You'll know the reaction is occurring when you spot tiny bubbles at the surface of the liquid a few hours after beginning the reaction.</p>
<p class="p7">Sadly for the Leuconostoc, they are too good at what they do. Eventually they'll consume all of the sugars they find transforming the liquid into a sour concoction, and as result the environment turns too acidic for them and they die.</p>
<p class="p7">By then, the conditions have become ideal for another species, <span class="s1">lactobacillus</span>, which was pretty much designed for acidic liquids, and they finish off the remaining sugars, until the acidic level reaches PH=3, at which point their work is done.</p>
<p class="p7">Fortunately for humans, the pathogens that could cause us harm are unable to survive in this high-acidic environment, and would be out-competed by the existing microbes anyway, so the final mixture is perfectly healthy.</p>
<p class="p8"><br></p>
<p class="p7"><b>More fermentation</b></p>
<p class="p7">Just about any food can be fermented, as people in all cultures have known since pre-history. Many common ethnic foods, such as miso (soybean) or kimchee (cabbage), are possible only thanks to fermentation. When you eat "cured" bacon or sausage, you are eating something that resulted from a complex interaction between microbes and meat.<span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<p class="p8"><span class="Apple-converted-space"> </span></p>
<p class="p8"><br></p>
<h1 style="margin: 0.0px 0.0px 0.0px 0.0px; font: 16.0px Georgia"><b><a id="doc59"></a>Annnotated References</b></h1>
<p class="p7">Here are my suggestions for additional reading.</p>
<p class="p8"><br></p>
<p class="p7">Read more about Toxoplasma Gondii: <a href="http://www.theatlantic.com/magazine/archive/2012/03/how-your-cat-is-making-you-crazy/308873/">http://www.theatlantic.com/magazine/archive/2012/03/how-your-cat-is-making-you-crazy/308873/</a></p>
<p class="p8"><br></p>
<p class="p8"><br></p>
<hr>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<p class="p2"><br></p>
<h1 style="margin: 0.0px 0.0px 24.0px 0.0px; text-align: center; font: 24.0px Georgia"><b><a id="doc29"></a>Endnotes</b></h1>
</div>
</body>
</html>
